src/tests/fake/ml.ts

```diff
@@ -0,0 +1,93 @@
+import { faker } from "@faker-js/faker";
+import { merge } from "lodash";
+import nock from "nock";
+import { MLModel, MLModelMinimal, MLModelRelease } from "../../js/ml/types";
+
+type CreateFakeMLModelReleaseProps = {
+    id?: number;
+    created_at?: string;
+    download_url?: string;
+    github_url?: string;
+    name?: string;
+    published_at?: string;
+    ready?: boolean;
+    size?: number;
+};
+
+/**
+ * Create a fake ML model release object
+ *
+ * @param props - optional properties for creating a fake ML models with specific values
+ */
+export function createFakeMLModelRelease(props?: CreateFakeMLModelReleaseProps): MLModelRelease {
```

Comment:
I find the 'props' naming confusing since this is not a component. The naming of the type could be especially confusing when using autocomplete or code search. Maybe `overrides`? I'll leave it to you.

Do we really need to allow any of the model fields to be overridden? I think we should have no overriding possible unless use cases arise.